### PR TITLE
docs: inline attribute project-name to Quarkus

### PR DESCRIPTION
**Pull request by rquinio**
Remove {project-name} attribute usage from the docs and replace it by `Quarkus`
This is a follow-up from a review comment in https://github.com/quarkusio/quarkus/pull/2913

### CODE REVIEW

File: docs/src/main/asciidoc/README.adoc [hunk 1]
```diff
@@ -1,8 +1,7 @@
 = How to Create Quarkus Documentation
 include::./attributes.adoc[]
 
-This guide describes the asciidoc format and conventions that the {project-name} has
-adopted.
+This guide describes the asciidoc format and conventions that the Quarkus has adopted.
```
Comment (gsmet) [inline docs/src/main/asciidoc/README.adoc]: hmm, this was wrong. It should either be `the Quarkus project` or just `Quarkus`

File: docs/src/main/asciidoc/infinispan-client-guide.adoc [hunk 2]
```diff
@@ -246,7 +246,7 @@ And you pass the marshaller by defining the following:
 ----
 
 NOTE: Annotation based proto stream marshalling is not yet supported in
-the {project-name} infinispan client.
+the Quarkus infinispan client.
```
Comment (gsmet) [inline docs/src/main/asciidoc/infinispan-client-guide.adoc]:
```suggestion
the Quarkus Infinispan client.
```
(preexisting but while we're at it...)

File: docs/src/main/asciidoc/infinispan-client-guide.adoc [hunk 3]
```diff
@@ -319,15 +319,15 @@ You can do this by adding the following dependency to your project.
 With this added you can use Infinispan querying just as you would normally, nothing special required.
 You can read more about this at https://infinispan.org/docs/dev/user_guide/user_guide.html#query_dsl.
 
-You can use either the Query DSL or the Ickle Query language with the {project-name} infinispan client
+You can use either the Query DSL or the Ickle Query language with the Quarkus infinispan client
```
Comment (gsmet) [inline docs/src/main/asciidoc/infinispan-client-guide.adoc]:
```suggestion
You can use either the Query DSL or the Ickle Query language with the Quarkus Infinispan client
```

File: docs/src/main/asciidoc/infinispan-client-guide.adoc [hunk 4]
```diff
@@ -319,15 +319,15 @@ You can do this by adding the following dependency to your project.
...
 extension.
 
 == Counters
 
-Infinispan also has a notion of counters and the {project-name} infinispan client supports them out of
+Infinispan also has a notion of counters and the Quarkus infinispan client supports them out of
```
Comment (gsmet) [inline docs/src/main/asciidoc/infinispan-client-guide.adoc]:
```suggestion
Infinispan also has a notion of counters and the Quarkus Infinispan client supports them out of
```

File: docs/src/main/asciidoc/infinispan-client-guide.adoc [hunk 5]
```diff
@@ -319,15 +319,15 @@ You can do this by adding the following dependency to your project.
...
 the box.
 
-The {project-name} infinispan client extension allows for Dependency Injection
+The Quarkus infinispan client extension allows for Dependency Injection
```
Comment (gsmet) [inline docs/src/main/asciidoc/infinispan-client-guide.adoc]:
```suggestion
The Quarkus Infinispan client extension allows for Dependency Injection
```

**Review by gsmet — COMMENTED** [review]
Nice cleanup, thanks.

I added a few comments about preexisting issues but while we're at it, could you fix them?

**Comment by rquinio** [comment]
Sure, done !

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**